packages/webapp/hooks/useWebappVersion.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useState } from 'react';
-import { useRouter } from 'next/router';
 import type { ParsedUrlQuery } from 'querystring';
 import { isIOSNative } from '@dailydotdev/shared/src/lib/func';
+import { useRouter } from 'next/router';
```

Comment:
Since it's just a change of order, could we move it back. One less change in the PR

Reply:
Done ✔️